code.json

```diff
@@ -88,6 +88,51 @@
       "metadataLastUpdated": "2023-04-19"
     }
   },{
+    "name": "ISIS3",
+    "organization": "U.S. Geological Survey",
+    "description": "GitHub code repository for the Integrated Software for Imagers and Spectrometers (ISIS)",
+    "version": "8.3.0",
```

Comment:
This is what we should probably decide on now. We either need to remove 8.0.0 and replace with 8.3.0 or remove the RC from dev's code.json. Right now the lambda is looking for the last element and removing _RC. If we go with the first option, I can update the lambda to find the version that doesn't have `_RC` or `_rc` in it.

Reply:
Disregard. Discussed to instead update lambda to look in alphanumeric order, excluding RCs.